pytorch/translation/wmt17-transformer-all-reduce/main.py

```diff
@@ -0,0 +1,480 @@
+"""Training GNMT for WMT14 Dataset
+
+This implements the machine translation benchmark tasks,
+# TODO add link to docs
```

Comment:
small todo?

Reply:
Yes waiting to merge the documentation to get the link